modules/bbp-core/src/main/resources/schemas/bbp/core/brainLocation/v0.1.0.json

```diff
@@ -0,0 +1,159 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpvect3dsh": "{{base}}/schemas/bbp/core/vector3d/v0.1.0/shapes/",
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "bbpqsh": "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+                "{{base}}/schemas/bbp/core/vector3d/v0.1.0",
+                "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "@type": "owl:Ontology",
+  "shapes": [
+    {
+      "@id": "this:BrainLocationShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain location shape definition",
+      "property": [
+        {
+          "path": "nsg:brainRegion",
+          "name": "Brain region",
+          "description": "Brain region",
+          "node": "bbptlotsh:BrainRegionOntologyTermShape",
+          "minCount": 1
+        },
+        {
+          "path": "nsg:coordinatesInBrain",
+          "name": "Coordinates in brain",
+          "description": "Coordinates in brain",
+          "node": "this:CoordinatesInBrainShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:coordinatesInSlice",
+          "name": "Coordinates in slice",
+          "description": "Coordinates in slice",
+          "node": "this:CoordinatesInSliceShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:subRegion",
```

Comment:
subRegion should be removed. All brain region infos should be attached to the brain region term.